src/lib/storage/devices_list.rb

```diff
@@ -98,15 +98,19 @@ def match?(element, attr, value)
           real_value = element.send(attr)
         rescue ::Storage::WrongNumberOfChildren
           # Checking for something that is not there
-          return false
+          return value.nil?
```

Comment:
I think it deserve comment this logic. So if it is nil, then it match to all values? looks a bit strange